### PR TITLE
Fix #15598 - text size too large

### LINKED ISSUE #15598: Text size too large?
https://demo.phpmyadmin.net/master-config/index.php?route=/database/sql&db=assigment3

The bold text seems too large / larger than normal.

pma

### PR DESCRIPTION
**Pull request by tlshifat**
### Description

Text size was too large. Edited css of legend to 13px

Fixes #15598

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by shifatbuet** [comment]
please proceed . minor change.

**Comment by williamdes** [comment]
@shifatbuet I think you can do the following commands
```
git checkout hotfix/text_size_15598
git reset --hard 12db01bfc355915eabe1a23b96e068cfc43d37d4
git cherry-pick 38ad60a6070ad56dfa6d70713e4e307d6585bb8e
git push --force
```

**Review by williamdes — APPROVED** [review]

**Comment by shifatbuet** [comment]
> 
> 
> @shifatbuet I think you can do the following commands
> 
> ```
> git checkout hotfix/text_size_15598
> git reset --hard 12db01bfc355915eabe1a23b96e068cfc43d37d4
> git cherry-pick 38ad60a6070ad56dfa6d70713e4e307d6585bb8e
> git push --force
> ```



> 
> 
> @shifatbuet I think you can do the following commands
> 
> ```
> git checkout hotfix/text_size_15598
> git reset --hard 12db01bfc355915eabe1a23b96e068cfc43d37d4
> git cherry-pick 38ad60a6070ad56dfa6d70713e4e307d6585bb8e
> git push --force
> ```

done as per your instruction .

**Comment by williamdes** [comment]
@sudo-bot Deploy pull-request

**Comment by williamdes** [comment]
image

**Review by williamdes — APPROVED** [review]
Solved, thank you @shifatbuet for your contribution !

**Comment by williamdes** [comment]
@mauriciofauth If you approve the change you can merge :)

**Review by rahgurung — CHANGES_REQUESTED** [review]
This solves the problem but doesn't solve the reason why the problem was there in first place. Actually in old fixed version font-size was already `1rem`. The problem here is we have to set 
```  
html {
    font-size: 82%;
}
```
so that rem can get the correct size from the root element. This fix (the one I'm suggesting) further creates few more UI bugs which can be easily fixed by changing the respective element's font-size to 1 rem or whatever was there in old fixed version. `rem` should be preferred for global styling.

File: themes/pmahomme/scss/_common.scss [hunk 1]
```diff
@@ -350,7 +350,7 @@ fieldset {
     -webkit-box-shadow: 3px 3px 15px #bbb;
     box-shadow: 3px 3px 15px #bbb;
     width: initial;
-    font-size: 1rem;
+    font-size: 13px;
```
Comment (mauriciofauth) [inline themes/pmahomme/scss/_common.scss]: We don't want to use the `px` unit here.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Comment by williamdes** [comment]
@gurrrung can you fix your work ?

**Comment by gurrrung** [comment]
This is fixed already, #15816. Thanks for your contribution.

**Pull request closed without merging**